### ISSUE TITLE
Is the conversion lossless?

### ISSUE THREAD

**Issue by mariomadproductions** [adibsurani/Mobius#1]
If not, could you add avi support?

**Comment by AdibSurani** [comment]
Yes, the default options are `-preset ultrafast -crf 0`, which is lossless. Since it pipes the information out to ffmpeg, you can also easily make it output any format that ffmpeg supports (such as avi).

**Comment by ghost** [comment]
This can probably be closed, maybe an FAQ would be worth setting up?

**Comment by mariomadproductions** [comment]
Seems like this information was already on the readme before I made this issue. I think its probably clear enough. But I don't mind if you want to leave this issue open.

**Comment by fmlatghor** [comment]
> Seems like this information was already on the readme before I made this issue. I think its probably clear enough. But I don't mind if you want to leave this issue open.

It is, yet people still open these types of issues, so maybe an FAQ would make things clear? I don't know, it's just a suggestion.
